### PR TITLE
Update production 

### PR DESCRIPTION
**Pull request by kinueng**

### CODE REVIEW

**Comment by kinueng** [comment]
Hi @aknguyen7, from your Cypress testing, was staging branch passing the tests? If so, lets update production

**Comment by aknguyen7** [comment]
yes, `staging` branch with Cypress testing has tests passing (no regression) here https://github.com/OpenLiberty/start.openliberty.io/pull/114

File: server.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
+		allowedOrigins="https://www.openliberty.io,https://openliberty.io,https://draft-openlibertyio.mybluemix.net,https://staging-openlibertyio.mybluemix.net,https://ui-draft-openlibertyio.mybluemix.net,https://draft-ui-openlibertyio.mybluemix.net,https://ui-staging-openlibertyio.mybluemix.net,https://staging-ui-openlibertyio.mybluemix.net,https://demo1-openlibertyio.mybluemix.net,https://demo2-openlibertyio.mybluemix.net,https://us-south.openliberty.io,https://us-east.openliberty.io,https://eu-gb.openliberty.io,https://eu-de.openliberty.io,https://au-syd.openliberty.io,https://ui-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud,https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud"
```
Comment (natalie-bernhard) [inline server.xml:33]: Should these changes be in the server.xml? (wasn't sure if this was something specific to staging that ended up in the PR) If they are supposed to be there, everything else looks good to me.

**Review by aknguyen7 — APPROVED** [review]
lgtm

**Comment by kinueng** [comment]
image
image

**Comment by kinueng** [comment]
image

**Comment by kinueng** [comment]
I tested the failing test case in https://github.com/OpenLiberty/start.openliberty.io/pull/116#issuecomment-1249804867 by hand and the server slash page loaded fine

**Pull request merged**